index.html

```diff
@@ -110,7 +110,7 @@ <h3 class="paragraph_header">Close to nature</h3>
     <section class="pink">
       <div class="flex section_wrapper max_width">
         <div class="left_side flex-j-center">
-          <div class="some_text constant_width">
+          <div class="post_text constant_width">
```

Comment:
I think it is a `post_container` -> a div wrapping single post content

Reply:
fixed than :)